README.md

```diff
@@ -246,7 +252,55 @@ Here the `WallGrasp` strategy success depend on the relative orientation of the
 
 ## Examples  <a name="examples"></a>
 
+### Planning Based on Bag File Input 
+This example was tested with ros melodic under Ubuntu 18.04. 
+
+This example builds on the ifco detection example on the vision repository.
+The example bag file of this scene can be found here: /nas/Videos/Vision/ifco_example.bag, for access contact a tub-member. 
+
+<img src="docs/planner_example.png" alt="Grasp" width="250" />
+
+```
+# You must have a roscore running in order for ROS nodes to communicate.
+roscore
+
+# Launches ros parameters (can be adapted if necessary).
+roslaunch planner_gui parameters.launch
+
+# start the ros win bridge for RBO setup
+roslaunch planner_gui rosWinBridge.launch
+
+# start the openni2_camera drivers 
+roslaunch openni2_launch openni2.launch depth_registration:=false
+
+# play bag file (the example bag file is saved on the tub-NAS (as stated above))
+rosbag play -l ifco_example.bag
+
+# visualize in rviz
+rosrun rviz rviz -d `rospack find ec_grasp_planner`/configs/ec_grasps_example4.rviz 
+
+# execute vision with the compute_ec_graph service
+rosrun ecto_rbo_yaml plasm_yaml_ros_node.py `rospack find ecto_rbo_yaml`/data/demo_ifco.yaml --debug --service 
+
+# launch the planner 
+rosrun ec_grasp_planner planner.py --rviz --file_output --robot_base_frame "camera_link"
+
+# publish a static coordinate transform for the camera
+rosrun tf static_transform_publisher 0 0 0 0 0 0 camera_link camera 0 
+```
+```
+# to call the planner service from command line, run:
+rosservice call /run_grasp_planner "{object_type: 'mango', grasp_type: 'Any', handarm_type: 'RBOHandP24_pulpyWAM', object_heuristic_function: 'Deterministic', angle_of_attack: 0.0, wrist_angle: 0.0}"
+```
+or
+```
+# to call the planner service and execute the grasp with the GUI, run:  
+roslaunch planner_gui gui.launch
+# in GUI select "Grasp Type": Any, type: F1 - F2 - F3 - F4 - 0
+```
+
 ### Planning Based on PCD Input  <a name="example1"></a>
```

Comment:
A similar example could be added to soma_utiles and here too, where we use the recorderd PCD (in bag) by the gui to replan the grasp (using both vision and ec_grasp_planner).

Reply:
I added this example